Header header

int64 pwm
string mode
